wgengine/magicsock/magicsock.go

```diff
@@ -2599,86 +2595,91 @@ func (c *Conn) initialBind() error {
 	return nil
 }
 
-func (c *Conn) listenPacket(ctx context.Context, network, host string, port uint16) (net.PacketConn, error) {
+// listenPacket opens a listener. Host must be "" or an IP address.
+func (c *Conn) listenPacket(network, host string, port uint16) (net.PacketConn, error) {
+	ctx := context.Background() // unused without DNS name to resolve
```

Comment:
fine for now, but we can clean up later and replace `host string` with `ip netaddr.IP` and say a zero netaddr.IP is like "" today.

Reply:
Will do in a follow-up.